docs/roadmap.md

```diff
@@ -0,0 +1,185 @@
+# Roadmap
+
+## Aim of the project
+
+* Enable application developers to connect their workloads to backing services **with little or no code changes**.
+
+* Enable backing service authors to expose binding information by adding metadata to the the representative Kubernetes objects **with little or no code changes**.
+
+* Provide a managed binding experience.
+
+The Service Binding Operator is being designed to be adopted by application developers and backing service providers with a **low barrier for entry**.
+
+
+# Key feature areas
+
+### Collection of binding information
+* Annotation "Decorators" in backing service resources.
+* Descriptor "Decorators" in backing service resources.
+* Duck type in backing service resources.
+* Service Composition.
+* Detection of binding resources.
+
+### Project of binding informattion
```

Comment:
nit - information

Reply:
I would also say `Projection`, not `Project`